js/background.js

```diff
@@ -49,7 +49,7 @@
   const OPTIONS_DEFAULTS = {
     theme: 'sunburst',
     font: 'Inconsolata',
-    fontSize: 'medium',
+    fontSize: 16,
```

Comment:
Using font sizes as int to easily manipulate. `16` becomes `16px`. Could use `em`s if preferred.